rocprim/CMakeLists.txt

```diff
@@ -76,13 +79,24 @@ set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR})
 
 # We need to install headers manually as rocm_install_targets
 # does not support header-only libraries (INTERFACE targets)
+if(rocprim_hc)
```

Comment:
`if(TARGET rocprim_hc)`. 

Everything else looks ok. We will test it tomorrow and merge if it works ok on our CI with `hcc`.

Reply:
thanks!